coralproject/cay
# Round two: reactive clauses (previously sentence)

# What? 

Reactive clauses in the search creator.

# How do I test this? 

Move the filters around to see the new behavior. Be sure to try all combinations of slider positions (unchanged, equal, both, min, max)
